src/app/services/comment.service.ts

```diff
@@ -7,15 +7,19 @@ export class CommentService {
   constructor() { }
 
 //finds a user once using the userId
-  findUser(userId) {
+  userRef(userId) {
     const result = firebase.database().ref(`/blog/users/${userId}`);
-    return result;
+    if(result){
+      return result;
+    }
   }
 //finds a user once using the userId
-  async findUserOnce(userId) {
+  async userOnceRef(userId) {
```

Comment:
The original reason you named this method findUserOnce() was totally valid. However, in light of our evolving naming conventions, let's just call it "getUser()" since it's really just another method that returns a set value one time.